src/styles/utils/_variable.scss

```diff
@@ -0,0 +1,14 @@
+$watch-size: 500px;
+$center-size: $watch-size / 20;
+$color-main: black;
+$color-second: red;
+
+@keyframes rotate {
+  from {
+    transform: rotate(0);
+  }
+
+  to {
+    transform: rotate(1turn);
+  }
+}
```

Comment:
i don't think that its right place for animation

Reply:
agree, isn't the best place, you can keep it in dial.scss

Reply:
done!